store/cache/cache.go

```diff
@@ -15,7 +15,7 @@ var (
 
 	// DefaultCommitKVStoreCacheSize defines the persistent ARC cache size for a
 	// CommitKVStoreCache.
-	DefaultCommitKVStoreCacheSize uint = 1000
+	DefaultCommitKVStoreCacheSize uint = 100000
```

Comment:
we discussed this in the osmosis discord, it would be good to get some benchmarks to see how this improves things.